docs/source/api/v2/servers_details.rst

```diff
@@ -0,0 +1,166 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-servers-details:
+
+*******************
+``servers/details``
+*******************
+Retrieves details of :ref:`tp-configure-servers`.
+
+
+``GET``
+=======
+:Auth. Required: Yes
+:Roles Required: None
+:Response Type:  Array
```

Comment:
This should mention somewhere that the response includes additional, top-level properties besides `response` or `alerts`, because from the "Response Structure" subsection under "How to Read this Documentation", the "Array" response type is defined:

> _"The fields in the field list refer to the keys of the objects in the `response` array."_

So based on that, people might think the response here has the format:

```json
{
	"response": [{
		"limit": 1000,
		"hostName": "test",
		"response": ["some objects in here"]
	}]
}
```

if they don't look at the response example carefully.